emperor/support_files/js/controller.js

```diff
@@ -156,86 +194,48 @@ EmperorController.prototype.render = function() {
 };
 
 /**
+ *
  * Helper method to assemble UI, completely independent of HTML template
+ *
  **/
 EmperorController.prototype.buildUI = function() {
+  //FIXME: This only works for 1 scene plot view
+  this.colorController = this.addTab(this.sceneViews[0].decViews,
+                                     ColorViewController);
 
-  this.$plotMenu.append("<div id='emperor-menu-tabs'></div>");
-  $('#emperor-menu-tabs').append("<ul><li><a href='#keys'>Key</a></li></ul>");
-  $('#emperor-menu-tabs').append("<div id='keys' class='emperor-tab-div'></div>");
-  $('#emperor-menu-tabs').tabs({heightStyle: 'fill'});
+  // We are tabifying this div, I don't know man.
+  this._$tabsContainer.tabs({heightStyle: 'fill'});
+};
+
+/**
+ *
+ * Helper method to resize the plots.
+ *
+ * @param {Array} [dvdict] Dictionary of DecompositionViews.
+ * @param {function} [viewConstructor] Constructor of the view controller.
+ *
+ **/
+EmperorController.prototype.addTab = function(dvdict, viewConstructor){
+  // nothing but a temporary id
+  var id = "" + Math.round(1000000 * Math.random());
```

Comment:
Can you add a comment about this being faster than #.tostring() or other ways and perhaps a link to something that holds that?